Models/Base/MetaData/POCO/X.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace Backend.Models.Base.MetaData.POCO
+{
+    public class X
+    {
+        [JsonPropertyName("name")] public string Name { get; set; }
+
+        [JsonPropertyName("type")] public string Type { get; set; }
+
+        [JsonPropertyName("limit")] public List<Limit> Limit { get; set; }
+    }
+}
```

Comment:
As @Kpaubert mentioned, this could be extracted to one class; `Axes`.

Reply:
I would rather it be [axis](https://www.merriam-webster.com/dictionary/axis), singular.

Reply:
Wise words, funny magic man!